admin/class-fastcgi-purger.php

```diff
@@ -168,4 +168,19 @@ public function purge_all() {
 
 	}
 
+	/**
+	 * Constructs the base url to call when purging using the `get_request` method.
+	 *
+	 * @return string
+	 */
+	private function purge_base_url() {
+		$parse = parse_url( site_url() );
+		$path  = trim( apply_filters( 'rt_nginx_helper_purge_url_path', 'purge' ), '/' );
```

Comment:
Since this filter is only for FastCGI cache only and to change purge suffix, I'd suggest filter name `rt_nginx_helper_fastcgi_purge_suffix`.

Please let me know your thought.

Reply:
👍